### PR TITLE
Apply hr api to user details #48

### PR DESCRIPTION
**Pull request by DmitryKus**
#48

### CODE REVIEW

File: js/src/features/human-resouce/components/Profile.tsx [hunk 1]
```diff
@@ -1,55 +1,87 @@
...
+              style={{ width: "100%" }}
+              placeholder="Enter your skill"
+              onChange={handleSkillChange}
+              defaultValue={employee.skills}
+              disabled={!canEdit}
+            />
+          </div>
+          <div className="user-profile__row">
+            <label>Birthday</label>
+            <DatePicker
```
Comment (kwoon) [inline js/src/features/human-resouce/components/Profile.tsx:73]: Maybe better to make DatePicker uncontrolled too like Select above?

File: js/src/features/human-resouce/epics.ts [hunk 2]
```diff
@@ -1,58 +1,195 @@
...
+  );
+
+export const updateEmployeeBirthdayEpic: RootEpic = (
+  action$,
+  state$,
+  { api },
+) =>
+  action$.pipe(
+    filter(isActionOf(updateEmployeeBirthdayAsync.request)),
+    debounceTime(500),
```
Comment (kwoon) [inline js/src/features/human-resouce/epics.ts:75]: Can you clarify why do you need debounce?

File: js/src/pages/UserDetails.tsx [hunk 3]
```diff
@@ -1,24 +1,201 @@
...
-  const { user, loading } = useSelector(userDetails);
-  const { data } = useSelector(profile);
+  const { employee, loading } = useSelector(employeeDetails);
+  const authId = useSelector(authUserId);
   const dispatch = useDispatch();
-  const userProp = user && user.id.toString() === id ? user : null;
-  const load = (id: string) => dispatch(fetchUserDetailsAsync.request(id));
+
+  const canEdit = authId === id;
+  const canSeeAdminStuff = canEdit;
```
Comment (kwoon) [inline js/src/pages/UserDetails.tsx:42]: Is it temporary solution?

Comment (DmitryKus) [reply js/src/pages/UserDetails.tsx:42]: Yes. I need check a user role here

File: js/src/features/human-resouce/epics.ts (hunk 2 above)
Comment (DmitryKus) [reply js/src/features/human-resouce/epics.ts:75]: If you very fast change date :D Yep, it can be deleted

Comment (kwoon) [reply js/src/features/human-resouce/epics.ts:75]: Understood. Stay it here

**Pull request merged**